gdal/port/cpl_http.h

```diff
@@ -86,8 +87,16 @@ typedef struct {
 
 } CPLHTTPResult;
 
+/*! @cond Doxygen_Suppress */
+typedef size_t (CPL_STDCALL *CPLHTTPWriteFunc)(void *pBuffer, size_t nSize, size_t nMemb, void *pWriteArg);
+/*! @endcond */
+
 int CPL_DLL   CPLHTTPEnabled( void );
-CPLHTTPResult CPL_DLL *CPLHTTPFetch( const char *pszURL, char **papszOptions);
+CPLHTTPResult CPL_DLL *CPLHTTPFetch( const char *pszURL, char **papszOptions,
```

Comment:
This part should be C only API, so no default arguments

Reply:
Fixed